tests/chaos/oadp/conftest.py

```
@@ -79,3 +82,243 @@ def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
+    yield _create_and_wait
+
+    # Cleanup
+    for backup in created_backups:
+        try:
+            backup.__exit__(None, None, None)  # noqa: FCN001
+        except Exception:
+            LOGGER.exception(f"Failed to cleanup backup {backup.name}")
+
+
+@pytest.fixture()
+def oadp_restore_start_factory(admin_client, oadp_backup_completed_factory, deleted_vm):
```

Ruff (0.14.10)

Unused function argument: `oadp_backup_completed_factory`

(ARG001)